packages/cli/src/init.ts

```diff
@@ -11,6 +11,20 @@ import packageJSON from '../package.json';
 import { fileExists } from './files';
 import { DEFAULT_MANIFEST_FILE, writeIntegrationManifest } from './manifest';
 
+function validateURLSlug(slug: string): boolean {
+    return /^[a-z0-9]+(?:-[a-z0-9]+)*$/.test(slug);
+}
+
+function validateIntegrationName(name: string) {
```

Comment:
Instead of creating a custom function, you should update `IntegrationManifestSchema` in `manifest.ts` to validate the `name` and reuse the same zod schema here

Something like:

```
const IntegrationNameSchema = z.string().regex(regex);
```